### ISSUE TITLE
Internet Access Required

### ISSUE THREAD

**Issue by antfok** [TheInformationLabNL/Venafi-Tableau-WDC#2]
Is there a way to not require Internet Access in order to take advantage of this solution. It looks like there are calls to 

        <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js [ajax.googleapis.com]"></script>
        <script src="https://connectors.tableau.com/libs/tableauwdc-2.3.latest.js [connectors.tableau.com]"></script>
 
Looking to see if there are options to work around this as my Venafi install doesn't have Internet access.

**Comment by mverstrepen** [comment]
Hi @antfok, you could download the js files from the urls and store them locally and adjust the code so it references you're local version. This should limit the requirement to connect to the internet.
